Core/include/Acts/Vertexing/NumericalTrackLinearizer.ipp

```diff
@@ -23,7 +23,7 @@ Acts::NumericalTrackLinearizer<propagator_t, propagator_options_t>::
 
   // Length scale at which we consider to be sufficiently close to the Perigee
   // surface to skip the propagation.
-  pOptions.targetTolerance = m_cfg.targetTolerance;
+  pOptions.surfaceTolerance = m_cfg.targetTolerance;
```

Comment:
I guess this might also need to change to keep it consistent?

Reply:
I think in this case it is actually just the target surface tolerance

Reply:
Ok I see